extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaInfoboxBuilderDialog.scss

```diff
@@ -0,0 +1,3 @@
+.InfoboxBuilder {
```

Comment:
if this rule is only for infobox builder in VE context could you make this selector more specific?